docs/examples/frameworks/pytorch/pytorch-lightning.ipynb

```diff
@@ -0,0 +1,488 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And see how it works"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "GPU available: True, used: True\n",
+      "TPU available: False, using: 0 TPU cores\n",
+      "LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]\n",
+      "/usr/local/lib/python3.6/dist-packages/pytorch_lightning/utilities/distributed.py:45: UserWarning: WORLD_SIZE environment variable (2) is not equal to the computed world size (1). Ignored.\n",
+      "  warnings.warn(*args, **kwargs)\n",
+      "initializing ddp: GLOBAL_RANK: 0, MEMBER: 1/1\n",
+      "/usr/local/lib/python3.6/dist-packages/torch/cuda/__init__.py:125: UserWarning: \n",
+      "GeForce GT 710 with CUDA capability sm_35 is not compatible with the current PyTorch installation.\n",
+      "The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_70 sm_75.\n",
+      "If you want to use the GeForce GT 710 GPU with PyTorch, please check the instructions at https://pytorch.org/get-started/locally/\n",
+      "\n",
+      "  warnings.warn(incompatible_device_warn.format(device_name, capability, \" \".join(arch_list), device_name))\n",
+      "\n",
+      "  | Name    | Type   | Params\n",
+      "-----------------------------------\n",
+      "0 | layer_1 | Linear | 100 K \n",
+      "1 | layer_2 | Linear | 33 K  \n",
+      "2 | layer_3 | Linear | 2 K   \n",
+      "WARNING: Logging before flag parsing goes to stderr.\n",
+      "I1014 17:16:18.776570 139719591925568 lightning.py:1215] \n",
+      "  | Name    | Type   | Params\n",
+      "-----------------------------------\n",
+      "0 | layer_1 | Linear | 100 K \n",
+      "1 | layer_2 | Linear | 33 K  \n",
+      "2 | layer_3 | Linear | 2 K   \n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "a6d6ed102f5f4bfe92e0f9361e33662f",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "HBox(children=(HTML(value='Training'), FloatProgress(value=1.0, bar_style='info', layout=Layout(flex='2'), max…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "1"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model = LitMNIST()\n",
+    "trainer = Trainer(gpus=1, distributed_backend=\"ddp\", max_epochs=5)\n",
+    "trainer.fit(model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let us define a DALI pipeline which would load the data."
```

Comment:
The next step is to define a DALI pipeline that will be used for loading and pre-processing data.

Reply:
Done